src/components/warning-text/template.test.js

```diff
@@ -0,0 +1,78 @@
+/* globals describe, it, expect */
+
+const { render, getExamples } = require('../../../lib/jest-helpers')
+
+const examples = getExamples('warning-text')
+
+describe('Warning text', () => {
+  it('renders the default example with text and assistive text', () => {
+    const { $ } = render('warning-text', examples.default)
+
+    const $component = $('.govuk-c-warning-text')
+    expect($component.text()).toContain('You can be fined up to £5,000 if you don’t register.')
+
+    const $assistiveText = $('.govuk-c-warning-text__assistive')
+    expect($assistiveText.text()).toEqual('Warning')
+  })
+
+  it('renders the default example with aria attribute for icon', () => {
```

Comment:
We could be clearer with our intent here. How about something like…

> It hides the icon from screen readers using the aria-hidden attribute

?

Reply:
A very good call 💯 Have amended.